src/app/components/ImportSourceBox.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+type Props = {
+  enabled: boolean;
+  logo: string;
+  title: string;
+  link: string;
+};
+
+//function DisableClick { (event) => event.preventDefault() };
+
+export function ImportSourceBox(props: Props) {
+  return (
+    <div className="my-5 px-5 w-1/4 overflow-hidden sm:w-1/3 md:w-1/2 lg:w-1/4 xl:w-1/4">
+      <Link
+        to={props.link}
+        className={`${
+          props.enabled == false && 'cursor-default'
+          } hidden md:flex justify-center`}
+        /*onClick={ props.enabled == false && {DisableClick} }*/
```

Comment:
Not sure how to disable click for disabled boxes

Reply:
So for `<button />` you can just add a `disabled` attribute. For this react-router link, I am not sure actually. Maybe do something like
```tsx
<Link
  to={props.enabled ? props.link : "/add-bookmark"}
```
in line 17 for now.